catalog/ESA/worldcover.libsonnet

```diff
@@ -0,0 +1,157 @@
+local subdir = 'ESA';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+
+{
+  local collection_id(version) = 'ESA/WorldCover/' + version,
+  local basename(version) = std.strReplace(collection_id(version), '/', '_'),
+  local base_filename(version) = basename(version) + '.json',
+  local self_ee_catalog_url(version) =
+    ee_const.ee_catalog_url + basename(version),
+  local self_url(version) = catalog_subdir_url + base_filename(version),
+
+  catalog_entry(version, citation):
+    {
+      stac_version: ee_const.stac_version,
+      type: ee_const.stac_type.collection,
+      stac_extensions: [
+        ee_const.ext_eo,
+        ee_const.ext_sci,
+        ee_const.ext_ver,
+      ],
+      id: collection_id(version),
+      title: 'ESA WorldCover 10m ' + version,
+      version: version,
+      'gee:type': ee_const.gee_type.image_collection,
+      description: |||
+        The European Space Agency (ESA) WorldCover 10 m 2020 product provides a
```

Comment:
v200 only has year 2021

Reply:
Done.

Reply:
Hardcoding a single year as the function parameter is overly specific because soon v200 will add 2020. I wouldn't bother passing it in - rather, in each concrete jsonnet file append the year-specific text to the shared description.

Reply:
I see, then yeah, a string makes more sense.